src/redux/usersPositions.ts

```diff
@@ -0,0 +1,358 @@
+import { AnyAction } from 'redux';
+import { uniswapClient } from '../apollo/client';
+import {
+  USER_HISTORY,
+  USER_MINTS_BUNRS_PER_PAIR,
+  USER_POSITIONS,
+} from '../apollo/queries';
+import { AppDispatch, AppGetState } from '@rainbow-me/redux/store';
+import { ethereumUtils, logger } from '@rainbow-me/utils';
+
+const PRICE_DISCOVERY_START_TIMESTAMP = 1589747086;
+
+export const priceOverrides = [
+  '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', // USDC
+  '0x6b175474e89094c44da98b954eedeac495271d0f', // DAI
```

Comment:
Why these 2 only? What happens with all the other USD pegged stablecoins?

Reply:
Actually, I have no idea. This part is just copied

Reply:
Let's leave like this I think